### DISCUSSION TITLE
CHCH-DT

### DISCUSSION THREAD

**Discussion by baliebox4916** [iptv-org/community#831] [Q&A]
### Channel ID

CHCHDT.ca

### Notes

Another Native Station from My Hometown Thats Like a Hybrid Between MeTV/Decades and both The CW and CNN Except a Canadian Version

Im Sure The People Behind this Station Wouldn't Mind

As it is Free on OTA Here and Their Transmitter (Which was the Same Transmitter That CING/Energy 108 Now the Energy 95.3 Transmitter) Can Reach all the Way to Toronto and Buffalo NY

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by trancer1994** [comment]
I think we already have it in the playlist. Sadly though I think the stream may be tokened.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19815
